bench/Micro.Benchmark/Micro.Benchmark.csproj

```diff
@@ -4,6 +4,8 @@
     <RuntimeFrameworkVersion>6.0.6</RuntimeFrameworkVersion>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <AssemblyName>Micro.Benchmark</AssemblyName>
+    <EnablePreviewFeatures>True</EnablePreviewFeatures>
```

Comment:
I'm using `Generic Math` here, so we have to use the preview feature.
@arek / @ppekrol - I don't _expect_ that to be an issue, but wanted your eyes on that as well.